src/DataViews/FrametrackDataView.cpp

```diff
@@ -2,23 +2,18 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#include "DataViews/ScopeDataView.h"
+#include "DataViews/FrametrackDataView.h"
 
 #include <vector>
 
 namespace orbit_data_views {
 
-uint64_t ScopeDataView::GetScopeIdFromRow(uint32_t row) const {
-  ORBIT_CHECK(row < indices_.size());
-  return indices_[row];
-}
-
-void ScopeDataView::OnEnableFrameTrackRequested(const std::vector<int>& selection) {
+void FrametrackDataView::OnEnableFrameTrackRequested(const std::vector<int>& selection) {
   metrics_uploader_->SendLogEvent(
       orbit_metrics_uploader::OrbitLogEvent::ORBIT_FRAME_TRACK_ENABLE_CLICKED);
 
   for (int i : selection) {
-    if (!IsScopeDynamicallyInstrumentedFunction(GetScopeIdFromRow(i))) continue;
+    if (!IsRowFunction(i)) continue;
```

Comment:
This is more or less how we've fixed this on release. Checking the value `GetFunctionInfoFromRow` for null is one of the possible implementations of `IsRowFunction`. But some DataViews should have a way to implement it differently. E.g. FunctionDataView has only functions, so it should always return `true`. `LiveFunctionsDataView` has access to the instance of `CaptureData`, that can provide ScopeInfos.

Reply:
Imagine, there scope_id, that is of type D, yet `GetFunctionInfoFromRow` returns null. It's certainly a bug, that would be more difficult to catch, if we were only checking the value of `GetFunctionInfoFromRow`.

Reply:
I didn't get this. Why do you want two ways of doing the same thing?

Reply:
Because at least semantics is different. 
GetFunctionInfoFromRow returns null. OK, why? Perhaps, the instance isn't constructed yet? Wait, are the symbols loaded? 
vs
Oh, IsRowFunction returns false. It's not a function!